### DISCUSSION TITLE
<title> Cannot Upload Files to Repo

### DISCUSSION THREAD

**Discussion by michaelhasey** [community/community#44487] [Repositories]
### Select Topic Area

Product Feedback

### Body

For some reason I am not able to upload files to my repo.  After commiting changes after file upload, the "processing your files..." message comes uand does not move past this.  File cannot be uploaded.  Tested various small files (image files) and nothing can be uploaded.  

Please help!

**Comment by aeromidd** [comment]
Same problem here. It seemed fine about an hour ago. Now as you say it just gets stuck on "Processing your files…This may take a few minutes to complete." - forever. I can't see anything in the workflows that's holding it up. Quite annoying. I'm glad it's not just me...

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @michaelhasey ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Gaerdil** [comment]
Hi!
I have exactly the same problem here.
At the end I even get a "Something went really wrong, and we can’t process that file. " error message.
It worked perfectly well just a few hours ago, but now I can't upload anything with the firefox web browser, even small files.
